src/bindings/lib/Tree_sitter_output.atd

```diff
@@ -32,6 +32,10 @@ type node = {
      tagged appropriately to avoid confusion. *)
   ~kind <ocaml default="Literal \"\"">: node_kind;
 
+  (* Missing nodes are suggested by tree-sitter. They don't exist in the
+     input. *)
+  ~is_missing: bool;
+
```

Comment:
The addition of a new field should not break older code unless it tries to pattern-match records exactly. I would be a problem only for pre-generated code (`semgrep-`* repos) but it's not the case.